{
  "id": "MAL-2025-2939",
  "summary": "Malicious code in asynctestpython (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2025-03-28T13:05:21Z",
  "published": "2025-03-28T13:05:21Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2025-03-28T13:05:21Z",
        "import_time": "2025-03-31T07:07:04.697413601Z",
        "versions": [
          "0.1"
        ],
        "id": "RLMA-2025-01937",
        "source": "reversing-labs",
        "sha256": "929e7aac59882b3f73eae18de1a5e7777a84a4711f64e8fab23136014a349d73"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "asynctestpython",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/asynctestpython"
      },
      "versions": [
        "0.1"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/asynctestpython/MAL-2025-2939.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}